ui/src/components/table/QTable.json

```diff
@@ -1007,6 +1007,79 @@
       }
     },
 
+    "row-expand": {
+      "desc": "Slot to define how a body row looks like; Suggestion: QTr + Td with `colspan=\"100%\"`",
```

Comment:
Slot to define how **the expanded part of a row** looks like; Suggestion: QTr + Td with `colspan=\"100%\"`